Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,21 @@
 function sum(elements) {
+  //it checks if the parameter is an array, if not it returns the message 'not an array'
+  if (!Array.isArray(elements)) return "not an array";
+  
+  //it filters out all non-numbers including NaN
+  const elementsFiltered = elements.filter((item) => typeof item ==="number"&& !Number.isNaN(item));
+  //if the array is empty it returns 0
+  if (elementsFiltered.length === 0) return 0;
```

Comment:
You have three special cases here - an empty array, a single-element array, and a multiple-element array. Can you think of how you could handle all three cases with just one set of instructions?